amgut/handlers/auth_handlers.py

```diff
@@ -42,7 +42,7 @@ def post(self):
             ag_login_id, skid, kitinfo['password'],
             kitinfo['swabs_per_kit'], kitinfo['verification_code'],
             printresults)
-        if success == -1:
+        if success != 1:
```

Comment:
The method can only return `1` and `-1`. Note, the method is a bit [concerning](https://github.com/biocore/american-gut-web/blob/master/scripts/ag#L104), any idea why there is a `commit` in the `except`?